CMakeLists.txt

```diff
@@ -9,6 +9,13 @@ set(DLIB_NO_GUI_SUPPORT OFF CACHE STRING "DLib GUI support" FORCE)
 set(DLIB_GIF_SUPPORT OFF CACHE STRING "DLib GIF support" FORCE)
 set(DLIB_JPEG_SUPPORT OFF CACHE STRING "DLib JPEG support"  FORCE)
 set(DLIB_PNG_SUPPORT OFF CACHE STRING "DLib PNG support"  FORCE)
+set(DLIB_USE_BLAS OFF CACHE STRING "DLib BLAS support"  FORCE)
+set(DLIB_USE_LAPACK OFF CACHE STRING "DLib LAPACK support"  FORCE)
+set(DLIB_USE_MKL_FFT OFF CACHE STRING "DLib MKL FFT support"  FORCE)
+set(DLIB_USE_MKL_SEQUENTIAL OFF CACHE STRING "DLib MKL Sequential support"  FORCE)
+set(DLIB_USE_MKL_WITH_TBB OFF CACHE STRING "DLib MKL with TBB support"  FORCE)
+
+
```

Comment:
Is this for faster building of dlib?

Reply:
Yup

Reply:
Fixed the .gitmodules link. These CMake Flags have to be merged to master to default certain options in DLIB while building.